src/components/CompareResults/RunValues.tsx

```diff
@@ -50,6 +50,10 @@ function RunValues(props: RunValuesProps) {
     setExpanded(!expanded);
   };
 
+  const onCopyValues = () => {
+    void navigator.clipboard.writeText(values.toString());
```

Comment:
I'm not fully sure of what we should copy. I was initially thinking separated by spaces (like how it is displayed) would have been better, but trying it with google spreadsheet convinced me that commas are a better option. So it's good to me!